src/commands/deploy/delete.ts

```diff
@@ -13,7 +13,8 @@ export async function run(client: FluorineClient, interaction: CommandInteractio
         await commands.fetch();
 
         if (name === 'all') {
-            await commands.set([]);
+            if (guildId) await commands.set([(await import('../../commands/deploy/index')).data.toJSON()]);
+            else await commands.set([]);
```

Comment:
```suggestion
            await commands.set([ guildId ? (await import('../../commands/deploy/index')).data.toJSON() : null ]);
```
not sure if it will work, try it and see™